app/Models/LeaderboardPhotos.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Gladiator\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class LeaderboardPhotos extends Model
```

Comment:
AH sorry i didn't catch this before. Convention is for models to be defined in the singular so this class should be `LeaderboardPhoto` instead